src/node/history.h

```diff
@@ -244,15 +248,17 @@ namespace ccf
 
     std::vector<uint8_t> to_v() const
     {
-      size_t vs =
-        sizeof(index) + sizeof(max_index) + root.SIZE + root.SIZE * path->sz;
+      size_t vs = sizeof(index) + sizeof(max_index) + root.h.size() +
+        root.h.size() * path->sz;
```

Comment:
Minor: Can you please bracket `(root.h.size() * path->sz);` while you're here? I spent a while staring at this wondering how it ever worked, but obviously `*` != `+`.